.changeset/odd-poets-complain.md

```diff
@@ -0,0 +1,5 @@
+---
+"@hashicorp/design-system-components": patch
+---
+
+`Dropdown` - Update the color of the text and icons in the selected state checkmark list item to match the styling of the others.
```

Comment:
Maybe clarify what "others" refers to. The other states I presume?

Reply:
Actually refers more to the other list items, which I've added some clarity on